docs/libertyExtensions.md

```diff
@@ -51,3 +54,15 @@ The following properties are supported for server configuration.
 | timeout | String | 1.0 | Waiting time before the server starts. The default value is 30 seconds. The unit is seconds. Only used with `libertyStart` and `deploy` tasks. | No |
 | var | Properties | 3.0 | Inline server variables that are written to the `configDropins/overrides/liberty-plugin-variable-config.xml` file in the server directory. The property name is used for the variable `name`, and the property value is used for the variable `value`.| No|
 | verifyAppStartTimeout | int | 2.0 | Wait time for checking message logs for start of all applications installed with the `deploy` task. Only used with the `libertyStart` task. Default value is 0 seconds with no verification. | No |
+
```

Comment:
I don't think we need to document these here again. You already documented them in the dev/devc task document. Just add something to line 8 above where it mentions compileJSP and installLiberty can be configured with their related extension in side the `liberty` block. Then remove the changes to lines 3 and 6 also.

Reply:
Done